src/azure-cli/azure/cli/command_modules/appconfig/_params.py

```diff
@@ -52,6 +53,11 @@ def load_arguments(self, _):
         type=int,
         help='Maximum number of items to return. Must be a positive integer. Default to 100.'
     )
+    identities_arg_type = CLIArgumentType(
+        nargs='*',
```

Comment:
Can we use nargs='+' ?

Reply:
To keep consistent with other cli commands about managed identity, we actually do want "*". Such that "az appconfig create --assign-identity" is allowed, and it will just use system assigned identity. an [example ](https://docs.microsoft.com/en-us/azure/storage/common/storage-encryption-keys-cli#assign-an-identity-to-the-storage-account)for storage account